src/index.md

```diff
@@ -43,7 +43,8 @@ You can run Renovate as:
 
 - an [Open Source npm package](https://www.npmjs.com/package/renovate)
 - a [pre-built Open Source image on Docker Hub](https://hub.docker.com/r/renovate/renovate)
-- a [free GitHub App](https://github.com/marketplace/renovate) that is hosted by [Mend](https://www.mend.io/)
 
-[Install our GitHub app now](https://github.com/marketplace/renovate){ .md-button .md-button--primary }
+Or you can use [the Mend Renovate App](https://github.com/marketplace/renovate) which is hosted by [Mend](https://www.mend.io/).
+
+[Install the Mend Renovate app now](https://github.com/marketplace/renovate){ .md-button .md-button--primary }
```

Comment:
The old button used `Install our GitHub app now`, which doesn't use the correct product name. But that button shows clearly it's for the GitHub platform only.

I wanted to add `(GitHub only)` to my improved button, but that would make it too big.

Do you have any ideas?

Reply:
I think mention it's for github in the same sentence somehow